packages/nimble-components/src/rich-text/models/mention-extension-configuration.ts

```diff
@@ -23,6 +40,13 @@ export class MentionExtensionConfiguration {
 
         this.viewElement = mentionInternals.viewElement;
         this.character = mentionInternals.character;
+        this.mappingConfigs = mentionInternals.mappingConfigs;
+        this.iconTemplate = mentionInternals.iconTemplate;
+        this.label = mentionInternals.buttonLabel ?? '';
```

Comment:
Propagate the name change for consistency. It makes it hard to follow when the name changes unnecessarily multiple times.


```suggestion
        this.buttonLabel = mentionInternals.buttonLabel ?? '';
```

Reply:
Updated